next/src/components/common/NavBar/NavMenu/NavMenuContent.tsx

```diff
@@ -12,9 +14,23 @@ type NavMenuContentProps = {
   colCount: number
   sections: MenuSection[]
   colorStyle: CSSProperties
+  parentLabel: string
+  parentHref: string
```

Comment:
I realised we have the "see all" link on OLO.

Please rewrite it so it's consistent with OLO. We tried with Kate to keep the NavBar as consistent as possible across projects. :)